node/pkg/telemetry/telemetry.go

```diff
@@ -67,16 +90,16 @@ func (enc *guardianTelemetryEncoder) EncodeEntry(entry zapcore.Entry, fields []z
 	bufCopy := make([]byte, len(buf.Bytes()))
 	copy(bufCopy, buf.Bytes())
 
-	// Convert the zapcore.Level to a logging.Severity
-	severity := logLevelSeverity[entry.Level]
+	// if skipPrivateLogs==true, then private logs don't go to telemetry
+	if enc.skipPrivateLogs {
+		if bytes.Contains(bufCopy, []byte("\"_privateLogEntry\":true")) {
```

Comment:
Couldn't this be tricked by adding this to some user-controlled input (either now or in the future)